gateway-api/api/app.js

```diff
@@ -0,0 +1,18 @@
+//app.js
+//Bootstrapping the express application
+//load all routes
+const app = require('express')();
+const cors = require('cors');
+const bodyParser = require('body-parser');
+
+//Allow Corss origin request
+app.use(cors());
+
+//Allow JSON request bodies
+app.use(bodyParser.json());
+
+//Load in routes
+app.post('/login',require('./login.js')());
```

Comment:
This file should not be here. This line should be included in the `app.js` in the main folder of the gateway api